### ISSUE TITLE
Identifies whether an offer has been executed with OfferCreate.

### ISSUE THREAD

**Issue by develoQ** [ripple/explorer#782]
Currently on the top page of explorer, all OfferCreate transactions are represented by blue circles.

I can't determine if the Offer object was simply created or if the Offer was executed on the DEX.

Although the transaction type is the same, it would be easier to understand the network activity if it were possible to determine whether or not an execution actually happened.

**Comment by ckniffen** [comment]
This is a good idea but will require parsing transaction meta data to determine the display which is now static based on `TransactionType`.  Will discuss with the design team what this will look like in addition to other transaction modifiers.

My main worry is this will complicate the design.  cc: @bugsbunnies

**Comment by bugsbunnies** [comment]
> IMO this should be added to the `Simple` page for the transaction, but shouldn't affect the display on the home page

I agree, the homepage is just meant to be an overview. If we were to add more detailed context it can get very messy and the homepage icons shouldn't be a place for that
